nautobot/extras/api/views.py

```diff
@@ -451,6 +450,67 @@ class JobModelViewSet(
     serializer_class = serializers.JobSerializer
     filterset_class = filters.JobFilterSet
 
+    # I don't see an easy way for swagger_auto_schema to provide a different schema depending on the API version.
+    # Since for now we default to the pre-1.3 API behavior, document that version in the schema for now.
+    @swagger_auto_schema(responses={"200": serializers.JobClassSerializer(many=True)})
+    def list(self, request, *args, **kwargs):
+        """List all known Jobs. Note that filtering of this list is only supported with API version 1.3 or later."""
+        if request.major_version > 1 or request.minor_version >= 3:
+            # API version 1.3 or later - standard model-based response
+            return super().list(request, *args, **kwargs)
```

Comment:
It seems fragile to have this hard-coded here. Safe to assume this is just stopgap while we come up with a viable version management strategy?

Reply:
Honestly I'm pretty comfortable with this approach for now. We'll have to have the code branch on API version **somewhere**, and right inside the view function itself seems like the most intuitive place for it. But I'm absolutely open to suggestions for refactoring if we do come up with something better.

Reply:
Eh it'll probably be a finer detail once we implement some sort of regimented versioning pattern. I expect the refactoring to happen at that time.